flake.lock

```diff
@@ -0,0 +1,38 @@
+{
+  "nodes": {
+    "flake-utils": {
+      "locked": {
+        "lastModified": 1609246779,
+        "narHash": "sha256-eq6ZXE/VWo3EMC65jmIT6H/rrUc9UWOWVujkzav025k=",
+        "owner": "numtide",
+        "repo": "flake-utils",
+        "rev": "08c7ad4a0844adc4a7f9f5bb3beae482e789afa4",
+        "type": "github"
+      },
+      "original": {
+        "owner": "numtide",
+        "repo": "flake-utils",
+        "type": "github"
+      }
+    },
+    "nixpkgs": {
+      "locked": {
+        "narHash": "sha256-WeuJZ0I5kjSa+xxxEMFNvBdlZCwktEKZBNexwWe6718=",
+        "path": "/home/teto/nixpkgs",
```

Comment:
This is not correct. please re-run `nix flake update --recreate-lock-file` after adding the line below.

Reply:
oops sry. Fixed